### PR TITLE
[CHI-329] Clicking drawer content should not close drawer component

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/187/ for more details.

**Comment by jllr** [comment]
@davidojedacl it would be best to make an approach to solve the issue similar to the popover element.
Can you change the code and mimic it, please?

Screen Shot 2019-07-16 at 6 24 26 PM

**Comment by davidojedacl** [comment]
done

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/190/ for more details.

File: src/chi/javascript/components/drawer.js [hunk 1]
```diff
@@ -50,12 +51,32 @@ class Drawer extends Component {
       self.toggle();
     };
 
-    this._closeClickEventListener = function() {
+    this._closeClickEventListener = function(e) {
+      e.preventDefault();
```
Comment (dani-cl-madrid) [inline src/chi/javascript/components/drawer.js]: @davidojedacl There should be no need to prevent default. Please remove and check it still works.

Comment (davidojedacl) [reply src/chi/javascript/components/drawer.js]: done

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/206/ for more details.

**Comment by mattnickles** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/232/ for more details.

**Pull request merged**